.github/CODEOWNERS

```diff
@@ -164,3 +164,9 @@ lerna.json @grafana/frontend-ops
 
 # Cloud middleware
 /grafana-mixin/ @grafana/hosted-grafana-team
+
+# Intent API
+/internal/coremodel/datasource/crd @grafana/cloud-middleware
+/internal/framework/kubecontroller @grafana/cloud-middleware
+/internal/intentapi @grafana/cloud-middleware
+/internal/k8sbridge @grafana/cloud-middleware
```

Comment:
Looks like this team doesn't have write access to this repo.

Reply:
mmm, we'll need to get that addressed - @torkelo, if we're going to have the cloud folks run with the k8s/intent api, can we grant access? Or perhaps we should make a new, smaller team than all of @'grafana/cloud-middleware?